src/model/core/EthqlBlock.ts

```diff
@@ -33,48 +38,111 @@ interface TransactionsRolesArgs extends WithTransactionFilter {
 }
 
 class EthqlBlock implements EthqlBlock {
```

Comment:
How can a class implement itself?

Reply:
Haha, welcome to Typescript. The class implements the interface with its same name, declared just above. It seems to be superfluous in this case because Typescript performs [Declaration Merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) anyway, but I feel it's good to clarify the relationship.